package.json

```diff
@@ -25,6 +25,8 @@
     "nuxt": "^2.16.1",
     "tailwind-heropatterns": "^0.0.8",
     "tailwindcss-aspect-ratio": "^3.0.0",
+    "vue": "2.7.16",
+    "vue-server-renderer": "2.7.16",
```

Comment:
Ik zou deze liever uit de code base houden

(prima als je het lokaal hebt omdat het daar anders niet werkte)

Reply:
@casperboone Komt omdat deze eigenlijk geinstalleerd word als dependency van nuxt toch?

Reply:
Yess

Reply:
@casperboone als ik ze weglaat dan failt de CI/CD pipeline helaas wel though :S

Reply:
Probably omdat ze nog wel in yarn.lock zullen staan, dus die moet even geüpdatet worden (of gerevert dus)